docs/docs/administration/reverse-proxy.md

```diff
@@ -64,3 +64,40 @@ Below is an example config for Apache2 site configuration.
    ProxyPreserveHost On
 </VirtualHost>
 ```
+
+### Traefik Proxy example config
+
+The example below is for Traefik version 3.
+
+The most important is to increase the `respondingTimeouts` of the entrypoint used by immich. In this example of entrypoint `websecure` for port `443`. Per default it's set to 60s which leeds to videos stop uploading after 1 minute (Error Code 499). In the example it's set to 10 minutes. Hence video-uploads will fail after 10 minutes instead of one.
+
+`traefik.yaml`
+```yaml
+[...]
+entryPoints:
+  websecure:
+    address: :443
+    # this section needs to be added
+    transport:
+      respondingTimeouts:
+        readTimeout: 0
```

Comment:
Looks like set to 0 here, which is no timeout, not 10 minutes

Can we also add writeTimeout?

Reply:
Hey, thanks for your comment. I don't agree with using a timeout of 0, all our other examples use 10 minutes which has worked well.

Reply:
Hey, thanks for your comment. I don't agree with using a timeout of 0, all our other examples use 10 minutes which has worked well. Regardless of what we pick, it's never good to depend on defaults - we should specify all 3 values as the default can change in a later release.

Reply:
> Would prefer 10 minutes like other examples

I agree. Same for idle timeout. I think it's bad practise to not terminate connections at any point. I'll change it back.

Reply:
Thanks, I don't know of other examples using 10 minutes. To maintain consistency with other documents, may use 10 minutes. In my configuration, I use a value of 0 so i just suggest it